### PR TITLE
Release 1.12.0(develop) 

### PR DESCRIPTION
**Pull request by marinafrydenb**
Version 1.12.0

*   Added information on 3-D Secure 2 for Checkout and 
    Card Payments. 
*   Updated reconciliation files in Settlement &
    Reconciliation.
*   Added test card for Forbrugsforeningen in Test data.
*   Documented `paid`, `failed` and `aborted` across all resources.
*   Added additional information on `payerReference`, `generateRecurrenceToken`,
    `paymentToken`, `generatePaymentToken`, `recurrenceToken` and
    `instrument` for the `paymentorder` resource in Checkout.
*   Corrected and updated `view-` operations for all resources. 
*   Updated Seamless View Events for all resources.
*   Described `nonPaymentToken` and `externalNonPaymentToken` in
    Card Payments.
*   Small corrections to Swish Payments documentation. 
*   Clarified the `updateOrder` documentation.

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375&bid=19038375)

File: resources/release-notes.md [hunk 1]
```diff
@@ -31,6 +31,26 @@ published on this page." %}
 body="The version numbers used in headers on this page refers to the version of
 this very documentation, not to a version of any APIs described by it." %}
 
+## July 2020
```
Comment (payex-dev) [inline resources/release-notes.md:34]: Codacy found an issue: [[no-empty-sections] Remove empty section: "July 2020"](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

File: resources/release-notes.md [hunk 2]
```diff
@@ -31,6 +31,26 @@ published on this page." %}
...
+
+## Version 1.12.0
+
+*   Added information on 3-D Secure 2 for [Checkout][checkout-3ds2] and 
+    Card Payments[card-3ds2].
```
Comment (asbjornu) [inline resources/release-notes.md:39]:
```suggestion
    [Card Payments][card-3ds2]. 
```

File: resources/release-notes.md [hunk 3]
```diff
@@ -31,6 +31,26 @@ published on this page." %}
...
 
+## July 2020
+
+## Version 1.12.0
+
+*   Added information on 3-D Secure 2 for [Checkout][checkout-3ds2] and 
+    Card Payments[card-3ds2]. 
+*   Updated reconciliation files in [Settlement &
+    Reconciliation][settlement-reconcilitation].
+*   Added testcard for Forbrugsforeningen in [Test data][test-data].
```
Comment (asbjornu) [inline resources/release-notes.md:42]:
```suggestion
*   Added test card for Forbrugsforeningen in [Test data][test-data].
```

File: resources/release-notes.md [hunk 4]
```diff
@@ -31,6 +31,26 @@ published on this page." %}
...
+*   Documented `paid`/`failed`/`aborted` operations for all Payments.
```
Comment (asbjornu) [inline resources/release-notes.md:43]:
```suggestion
*   Documented `paid`, `failed` and `aborted` across all resources.
```

File: resources/release-notes.md [hunk 5]
```diff
@@ -31,6 +31,26 @@ published on this page." %}
...
+
+*   Added information on 3-D Secure 2 for [Checkout][checkout-3ds2] and 
+    Card Payments[card-3ds2]. 
+*   Updated reconciliation files in [Settlement &
+    Reconciliation][settlement-reconcilitation].
+*   Added testcard for Forbrugsforeningen in [Test data][test-data].
+*   Documented `paid`/`failed`/`aborted` operations for all Payments. 
+*   Added additional information on `payerReference`, `generateRecurrenceToken`,
+    `paymentToken`, `generatePaymentToken`, `recurrenceToken` and
+    `instrument`for `PaymentOrder` operations in [Checkout][checkout].
```
Comment (asbjornu) [inline resources/release-notes.md:46]:
```suggestion
    `instrument` for the `paymentorder` resource in [Checkout][checkout].
```

File: resources/release-notes.md [hunk 6]
```diff
@@ -31,6 +31,26 @@ published on this page." %}
...
+*   Corrected and updated `view`- operations for all Payment instruments.
```
Comment (asbjornu) [inline resources/release-notes.md:47]:
```suggestion
*   Corrected and updated `view-` operations for all resources. 
```

File: resources/release-notes.md [hunk 7]
```diff
@@ -426,6 +449,7 @@ integration and the payer.
 [swish-other-features]: /payments/swish/other-features
 [swish-redirect]: /payments/swish/redirect
 [swish-seamless-view]: /payments/swish/seamless-view
+[seamless-view-events-card]: payments/card/other-features#seamless-view-events
```
Comment (asbjornu) [inline resources/release-notes.md:452]: Codacy is saying this link doesn't work.

File: resources/release-notes.md [hunk 8]
```diff
@@ -31,6 +31,26 @@ published on this page." %}
...
+    Card Payments[card-3ds2]. 
+*   Updated reconciliation files in [Settlement &
+    Reconciliation][settlement-reconcilitation].
+*   Added testcard for Forbrugsforeningen in [Test data][test-data].
+*   Documented `paid`/`failed`/`aborted` operations for all Payments. 
+*   Added additional information on `payerReference`, `generateRecurrenceToken`,
+    `paymentToken`, `generatePaymentToken`, `recurrenceToken` and
+    `instrument`for `PaymentOrder` operations in [Checkout][checkout].
+*   Corrected and updated `view`- operations for all Payment instruments. 
+*   Updated [Seamless View Events][seamless-view-events-card] for all Payments.
```
Comment (asbjornu) [inline resources/release-notes.md:48]:
```suggestion
*   Updated [Seamless View Events][seamless-view-events-card] for all resources.
```

File: resources/release-notes.md [hunk 9]
```diff
@@ -31,6 +31,26 @@ published on this page." %}
...
+*   Added `nonPaymentToken` and `externalNonPaymentToken` description in
```
Comment (asbjornu) [inline resources/release-notes.md:49]:
```suggestion
*   Described `nonPaymentToken` and `externalNonPaymentToken` in
```

File: resources/release-notes.md (hunk 7 above)
Comment (marinafrydenb) [reply resources/release-notes.md]: corrected!

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375&bid=19038375)

File: resources/release-notes.md [hunk 10]
```diff
@@ -41,27 +61,37 @@ this very documentation, not to a version of any APIs described by it." %}
...
 *   Added a new section [Co-badge Card Choice for Dankort][co-badge-card] in
     [Card Payments][card-payment-url].
 
+## 07 May 2020
+
 ### Version 1.9.2
 
 *   Corrected table in [payments page][payments]
 
+## 07 May 2020
```
Comment (payex-dev) [inline resources/release-notes.md:93]: Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (87:1)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

File: _includes/card-authorization-transaction.md [hunk 11]
```diff
@@ -136,8 +136,7 @@ Content-Type: application/json
...
-| └➔&nbsp;`externalNonPaymentToken` | `string`  | Result of external tokenization. This value varies depending on cards, acquirer, customer, etc. For ICA cards, the token comes in response from Swedbank. For Mass Transit(SL) it is populated with PAR if it comes in response from the redeemer (Visa). If not, our own token (Mastercard / Amex). |
-|                                   |
+| └➔&nbsp;`externalNonPaymentToken` | `string`  | Result of external tokenization. This value varies depending on cards, acquirer, customer, etc. For ICA cards, the token comes in response from Swedbank. For Mass Transit(SL) it is populated with PAR if it comes in response from the redeemer (Visa). If not, our own token (Mastercard / Amex).                                                                                                                                                                 |
```
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:139]: Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375&bid=19038375)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375&bid=19038375)

File: _includes/card-authorization-transaction.md (hunk 11 above)
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:139]: Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375&bid=19038375)

File: _includes/card-authorization-transaction.md (hunk 11 above)
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:139]: Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375&bid=19038375)

File: _includes/card-authorization-transaction.md (hunk 11 above)
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:139]: Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375&bid=19038375)

File: _includes/card-authorization-transaction.md (hunk 11 above)
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:139]: Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375&bid=19038375)

File: _includes/card-authorization-transaction.md (hunk 11 above)
Comment (payex-dev) [inline _includes/card-authorization-transaction.md:139]: Codacy found an issue: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5815375)

**Pull request merged**